android/build.gradle

```diff
@@ -23,6 +47,15 @@ android {
         targetSdkVersion safeExtGet('targetSdkVersion', 26)
         versionCode 1
         versionName "1.0"
+
+        def appId = heapAppId()
+        if (appId) {
+            buildConfigField("String", "HEAP_APP_ID", "\"${appId}\"")
```

Comment:
`"String"` and `"HEAP_APP_ID"` should probably be pulled out into a constant (more so the latter than the former, I think)... the goal would be just to reduce the number of accidental mistakes someone could make by fat-fingering a known string as they type it in... both for yourself (and I think we can both clearly see that you got it right) and for the next person!

Reply:
My intuition was that `resValue`s are more for UI-visible text/etc., while `BuildConfig` fields are for internal dispatch. Why/when would you pick one of these over the other?